components/usage/pkg/db/workspace_instance_usage.go

```diff
@@ -37,3 +38,14 @@ func CreateUsageRecords(ctx context.Context, conn *gorm.DB, records []WorkspaceI
 
 	return db.CreateInBatches(records, 1000).Error
 }
+
+func GetUsage(ctx context.Context, conn *gorm.DB, attributionId AttributionID) ([]WorkspaceInstanceUsage, error) {
```

Comment:
```suggestion
func ListUsage(ctx context.Context, conn *gorm.DB, attributionId AttributionID) ([]WorkspaceInstanceUsage, error) {
```

It returns a collection of records. Get is often used to mean a single record.